library/src/main/java/com/airbnb/android/airmapview/LeafletWebViewMapFragment.java

```diff
@@ -0,0 +1,42 @@
+package com.airbnb.android.airmapview;
+
+import java.util.Locale;
+
+import com.google.android.gms.maps.model.LatLng;
+
+public class LeafletWebViewMapFragment extends WebViewMapFragment {
+  public static LeafletWebViewMapFragment newInstance(AirMapType mapType) {
+    return (LeafletWebViewMapFragment) new LeafletWebViewMapFragment().setArguments(mapType);
+  }
+
+  @Override
+  public void setMapType(MapType type) {
+    String webType;
+    if (type == MapType.MAP_TYPE_NORMAL) {
```

Comment:
you can get rid of this first `if` and just let it fall through the `else` or use a `switch`

Reply:
Make sense. I'll use `switch` instead in all map fragments.